phys2bids/io.py

```diff
@@ -69,18 +68,22 @@ def check_multifreq(timeseries, freq, start=0, leftout=0):
     return mfreq
 
 
-def process_labchart(channel_list, chtrig, header=[]):
+def process_blueprint_items(channel_list, chtrig, interval, orig_units, orig_names):
```

Comment:
Since this function (1) does some processing on inputs and (2) generates a BlueprintInput object, why not just `generate_blueprint()`?